### ISSUE TITLE
Can't use Translate API

### ISSUE THREAD

**Issue by HamzaLJ** [driftyco/ionic#7696]
I would like to know how to user translate pipe built in Ionic2:

http://ionicframework.com/docs/v2/api/translation/Translate/

What to import, form where, where is the .json file that include the translated words, etc...

The documentation is not clear at all, so I was not able to implement it.

I know, that's maybe not a good place to post such thread, but I really need some help on how to do that.

Can you please some information on how to do it?

**Comment by NoNameProvided** [comment]
From #7691 

> The reason this has not been developed is because we are currently waiting on the official Angular 2 way of doing localization, and did not want to write our own implementation as wed rather use the Angular 2 way of doing things for localization.

I think for now you should go with some third-party stuff like [ocombe/ng2-translate](https://github.com/ocombe/ng2-translate). We use this at the moment, and it does the job.

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! I am going to be closing this issue as a duplicate of #7691 . Thanks for using Ionic!
